tests/functional/testMultihop.js

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ const { beforeEach } = require('mocha');
```

Comment:
empty line between line 3 and 4.

Reply:
plus this line can be removed. See other tests.

Reply:
ah thought i removed this already, good catch